Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -1695,6 +1702,27 @@ private void UpdateRigHandles()
 
                 // Compute the local scale that produces the desired world space dimensions
                 Vector3 linkDimensions = Vector3.Scale(GetLinkDimensions(), invRootScale);
+                Vector3 ballDimenions = Vector3.Scale(rotationHandleDiameter * Vector3.one, invRootScale);
+
+                Vector3 cornerDimensions = Vector3.Scale(cornerVisualScaleToMatchHandleSize * Vector3.one, invRootScale);
+                for (int i = 0; i < corners.Count; i++)
+                {
+                    if (scaleHandlePrefab == null)
+                    {
+                        corners[i].localScale = cornerDimensions;
+                    }
+                    else
+                    {
+                        // If we are modifying a corner that has a prefab, it has a different structure
+                        // Modify the visuals root instead.
+                        corners[i].GetChild(0).GetChild(0).localScale = cornerDimensions;
```

Comment:
this is not very clear to me. can there potentially be another prefab used that has a different hierarchy structure? can that break the behavior?

Reply:
I tried corners[i], and then corners[i].GetChild(0), but neither of those worked. The prefab is constructed in that very particular way, if you look at CreateScaleCorners(). You need to scale those particular nodes. I could instead create a list of the nodes to scale and use those...

Reply:
i was also quite bummed that i had to special case corners made via prefabs in this way. And then to look 2 levels deep in the hierarchy. Maybe @luis-valverde-ms  has some suggestions since he did bounding box prefabs?

Reply:
As far as i can remember I ended up with that hierarchy as the simplest one that would allow me to use mirroring scale to instance the same prefab for each corner.

Reply:
All right, I updated the approach to use an array of cornerVisual transforms, hopefully that approach is more understandable.